editor/src/core/performance/performance-regression-tests.spec.tsx

```diff
@@ -116,7 +116,7 @@ describe('React Render Count Tests - ', () => {
     )
 
     const renderCountAfter = renderResult.getNumberOfRenders()
-    expect(renderCountAfter - renderCountBefore).toBeGreaterThanOrEqual(660) // if this breaks, GREAT NEWS but update the test please :)
-    expect(renderCountAfter - renderCountBefore).toBeLessThan(670)
+    expect(renderCountAfter - renderCountBefore).toBeGreaterThanOrEqual(570) // if this breaks, GREAT NEWS but update the test please :)
```

Comment:
hmm, I presume this tests runs with a 0-high window, which means if we make a regression in the navigator render count, that will not be visible in a change here